font-icons/examples-icons.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Examples Icons
+page_title: .NET MAUI Font Icons - Overview
+description: Telerik font icons are collections of small vector graphics used across the components in the Telerik UI for .NET MAUI suite.
```

Comment:
Can you revise to provide some additional information about the list of icons people will see here?